zone/bot.cpp

```diff
@@ -3824,23 +4291,37 @@ void Bot::PerformTradeWithClient(int16 begin_slot_id, int16 end_slot_id, Client*
 			return;
 		}
 
-		if (trade_instance->IsStackable() && trade_instance->GetCharges() < trade_instance->GetItem()->StackSize) { // temp until partial stacks are implemented
-			if (trade_event_exists) {
-				event_trade.push_back(ClientTrade(trade_instance, trade_index));
-				continue;
-			}
-			else {
-				client->Message(
-					Chat::Yellow,
-					fmt::format(
-						"{} is only a partially stacked item, the trade has been cancelled!",
-						item_link
-					).c_str()
-				);
-				client->ResetTrade();
-				return;
+		if (RuleI(Bots, StackSizeMin) != -1) {
+			if (trade_instance->IsStackable() && trade_instance->GetCharges() < RuleI(Bots, StackSizeMin)) { // temp until partial stacks are implemented
+				if (trade_event_exists) {
+					event_trade.push_back(ClientTrade(trade_instance, trade_index));
+					continue;
+				}
+				else {
```

Comment:
The `else` here is not necessary because you `continue` when the previous condition is true